src/main/java/com/johnstarich/moviematcher/models/User.java

```diff
@@ -87,6 +87,7 @@ public User register(String password) throws HttpException {
 		}
 
 		User u = new User(id, username, first_name, last_name, friends, groups, BCrypt.hashpw(password, BCrypt.gensalt()));
+		// need to return session
```

Comment:
Why should we return a session here? This creates the user in the database, but does not log them in. That happens in the API route.